lib/spree_social/engine.rb

```diff
@@ -39,8 +40,21 @@ def self.init_provider(provider)
   end
 
   def self.setup_key_for(provider, key, secret)
+    scope = get_scope_for(provider)
+
     Devise.setup do |config|
-      config.omniauth provider, key, secret, setup: true, scope: 'email', info_fields: 'email, name'
+      config.omniauth provider, key, secret, setup: true, scope: scope, sandbox: true, info_fields: 'email, name'
```

Comment:
@nishant-cyro actually its required but it should be environment specific. 'sandbox' mode is needed for development environment in paypal auth.